src/site/_collections/authors.js

```diff
@@ -38,37 +38,14 @@ module.exports = (collections) => {
 
   Object.keys(authorsData).forEach((key) => {
     const authorData = authorsData[key];
```

Comment:
If I understand correctly, it seems like the author's title and description are removed from here, and instead moved into the addFields hook and added at a later time, is that correct? If so, why do it that way?

Reply:
The title and description are not removed here per se, they used to exist in the `authorsData.json` and no longer exist there. Instead further down the pipeline we'll just use the key of the author and look up their name/title and description using our i18n filter. So we know an authors title will always be `i18n.authors.${key}.title`, so when we need the title we just pass that into the filter.